### PR TITLE
Added signup screen

### PR DESCRIPTION
**Pull request by alealvarez14**
This PR adds:

- Signup screen
- Validations for name and gender
- Created SelectInput component

Preview:

screen shot 2018-10-22 at 6 15 07 pm

screen shot 2018-10-23 at 9 25 03 am

### CODE REVIEW

File: src/components/common/SelectInput.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  label,
+  values,
+  meta: { touched, error }
+}) => (
+  <div>
+    {label && <label>{label}</label>}
+    <div className="options-input">
+      <select className="fields" {...input} {...{ label }}>
+        <option value="" disabled="true">{defaultLabel}</option>
+        { values.map((item, index) => (<option key={index}> {item} </option>))}
```
Comment (Alxzu) [inline src/components/common/SelectInput.js]: Leave a space before last `}`

Comment (agch-dev) [reply src/components/common/SelectInput.js]: Actually it should be the other way around, there should be no space after the frist `{`

File: src/components/common/SelectInput.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  input,
+  defaultLabel,
+  label,
+  values,
+  meta: { touched, error }
+}) => (
+  <div>
+    {label && <label>{label}</label>}
+    <div className="options-input">
+      <select className="fields" {...input} {...{ label }}>
```
Comment (agch-dev) [inline src/components/common/SelectInput.js:17]: could you try doing: `{ ...input, label }`?

File: src/components/user/SignUpForm.js [hunk 3]
```diff
@@ -31,7 +44,8 @@ const SignUpForm = ({ handleSubmit, submitting, intl }) => (
     <div>
       <Field
         name="password"
-        label={intl.formatMessage(messages.password)}
+        placeholder="Min. 6 characters long"
+        label="Password" {...intl.formatMessage(messages.password)}
```
Comment (agch-dev) [inline src/components/user/SignUpForm.js]: put `{...intl.formatMessage(messages.password)}` in a new line.

**Review by agch-dev — COMMENTED** [review]
Great work! just few comments

**Review by agch-dev — APPROVED** [review]

**Review by Alxzu — APPROVED** [review]

**Pull request merged**